go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/tektoncd/pipeline
 
 go 1.22
+toolchain go1.22.4
```

Comment:
This is added because of https://github.com/jenkins-x/go-scm/blob/main/go.mod#L53 I think. We need to fix the `go-license` error I guess now.